### ISSUE TITLE
Arabic translate RTL

### ISSUE THREAD

**Issue by SajjadRahim** [blawar/translations#16]
Hi, I just installed latest version and i noticed that texts are left to right while it should be right to left. 

For example ﺕﺪﺍﺪﻉﺇ it should be إعدادات.
And its clearly unreadable for users
If you want help to fix it i can help

**Comment by blawar** [comment]
so I just need to reverse all of the characters in arabic?

**Comment by SajjadRahim** [comment]
Yeah, you can reverse characters by using tadween: https://sourceforge.net/projects/tadween

let me know if need to check anything.

Thanks.

**Comment by blawar** [comment]
RTL should be fixed in 7.0, however there are certain glyphs will not render.  apparently arabic has some complicated glyph rendering where some characters reference other characters, i need to figure it out but it should be more readable now :)

**Comment by SajjadRahim** [comment]
well done... i just tested now and it's seem the font you used missing Isolated
form in Arabic Presentation Forms-B and maybe this why glyphs not render because it's empty in first place.
